config/jobs/kubernetes/cloud-provider-aws/cloud-provider-aws-periodics.yaml

```diff
@@ -0,0 +1,87 @@
+periodics:
+- interval: 6h
+  cluster: k8s-infra-prow-build
+  name: ci-cloud-provider-aws-e2e
+  decorate: true
+  decoration_config:
+    timeout: 80m
+  annotations:
+    testgrid-tab-name: E2E - Cloud Provider AWS
+    testgrid-dashboards: provider-aws-periodics
+    description: Runs e2e against cloud-provider-aws master on AWS
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+    preset-aws-credential-aws-oss-testing: "true"
+    preset-k8s-ssh: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: cloud-provider-aws
+    base_ref: master
+    path_alias: k8s.io/cloud-provider-aws
+    workdir: true
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230727-ea685f8747-master
+      resources:
+        limits:
+          cpu: 4
+          memory: 14Gi
+        requests:
+          cpu: 4
+          memory: 14Gi
```

Comment:
do you need so much resources?

Reply:
sorry copy-pasta from the gcp job.

looks like the presubmit does not specify any, will mirror the same.

Reply:
I know kind jobs require them or it flake, but I'm not familiar with others